setup.py

```diff
@@ -12,25 +10,32 @@
 
 
 def _run(cmd: list, **kwargs):
-    p = subprocess.run(cmd, **kwargs)
-    p.check_returncode()
-    return p
+    cmd = ' '.join(cmd)
+    print(f'Now running: {cmd}')
+    kwargs['shell'] = True
+    kwargs['stdout'] = subprocess.PIPE
+    kwargs['stderr'] = subprocess.PIPE
+    p = subprocess.Popen(cmd, **kwargs)
+    stdout, stderr = p.communicate()
+    if p.returncode:
+        print(f'\nstdout: {stdout}\n')
+        print(f'\nstderr: {stderr}\n\n')
+        raise subprocess.CalledProcessError(p.returncode, cmd, stdout, stderr)
```

Comment:
Please move the subprocess refactors into a separate PR.

Also, please use `run` instead of `Popen`. This package requires at least Python 3.7, and `run` provides stdout and stderr with a simpler API.